observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,72 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+// note to self: update errata live ID
+* link:https://access.redhat.com/errata/RHSA-2024:0853[Network Observability Operator 1.6.0]
+
+[id="network-observability-operator-1.6.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-lokiless-enhancements-1.6"]
+==== Enhanced use of Network Observability Operator without Loki
+You can now do more than ever with the Network Observability Operator by using Prometheus metrics and relying less on Loki for storage. For more information, see xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki].
+
+[id="network-observability-custom-metrics-1.6"]
+==== Custom metrics API
+You can create custom metrics out of flowlogs data by using the `FlowMetrics` API. Flowlogs data can be used with Prometheus labels to customize cluster information on your dashboards. You can add custom labels for any subnet that you would want to identify in your flows and metrics. This enhancement can also be used to better identify traffic from or to IPs that are external to the cluster. For more information, see xref:../../observability/network_observability/metrics-alerts-dashboards.adoc#network-observability-custom-metrics_metrics-dashboards-alerts[Custom metrics] and <FLOW_METRICS_API_REF_LINK_HERE>
+
+[id="network-observability-eBPF-performance-enhancements-1.6"]
+==== eBPF performance enhancements
+With the following updates, you can experience improved performances of the eBPF agent, both in terms of CPU and memory: 
+
+* The eBPF agent now uses TCX webhooks instead of TC.
+* The *NetObserv / Health* dashboard integration with eBPF metrics allows you to view alerts in two cases: when the Network Observability eBPF agent hashmap table is full, which causes the eBPF agent processes flows with degraded performance, or when the capacity limiter is triggered. 
```

Comment:
```suggestion
* Now see alerts using the *NetObserv / Health* dashboard integration with eBPF metrics in the following two cases: 

** When the Network Observability eBPF agent hashmap table is full, which causes the eBPF agent processes flows with degraded performance.
** When the capacity limiter is triggered. 
```

Reply:
In fact there's 2 different things here:
- the *NetObserv / Health* dashboard has a new section showing eBPF metrics
- based on the new eBPF metrics, an alert is created to notify when the eBPF agent is dropping flows.

Perhaps I wouldn't go in the details of what could cause the alert here? (just saying "when the agent is dropping flows" should be sufficient for the release note?